arm-logic/2016-06-01/examples/SessionsCreateOrUpdate.json

```diff
@@ -0,0 +1,40 @@
+{
+  "title": "Create or update integration account session example",
+  "parameters": {
+    "api-version": "2016-06-01",
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "integrationAccountName": "testia123",
+    "sessionName": "testsession123-ICN",
+    "session": {
+      "properties": {
+        "content": "{\"controlNumber\":\"1234\",\"controlNumberChangedTime\":\"2017-02-21T22:30:11.9923759Z\"}"
```

Comment:
The session content property is an opaque JToken for the RP. The clients define what they want to store in it. controlNumber is only one of the possible content and its schema is subject to change with client updates. The RP does not know the schema(s).